test/modules/XMLHttpRequest/compliance/xhr-event-handlers.browser.test.ts

```diff
@@ -0,0 +1,55 @@
+import { HttpServer } from '@open-draft/test-server/http'
+import { test, expect } from '../../../playwright.extend'
+import { useCors } from '../../../helpers'
+
+const httpServer = new HttpServer((app) => {
+  app.use(useCors)
+  app.get('/resource', (req, res) => {
+    res.status(200).send('hello')
+  })
+})
+
+test.beforeAll(async () => {
+  await httpServer.listen()
+})
+
+test.afterAll(async () => {
+  await httpServer.close()
+})
+
+test('onloadend handler is called', async ({ page, loadExample }) => {
```

Comment:
Maybe let's specify it a bit more explicitly in the test name that this concerns the scenario when _not_ returning a mocked response?

Reply:
Good point. Do you think that’s good enough?

Reply:
I like it. Thanks!